src/certified_datapacks/requirement.md

```diff
@@ -1,11 +1,27 @@
-# Requirement
-
-A datapack must satisfy these requirements to receive the Certified Datapack title
-
-- Datapack must be portable, it must not be tied down to a specific save.
-- Datapack must follow all of **Tier 3** Conventions.
-
-## Note
-
-1. We may rule a datapack out at our discretion, for reasons we would then specify.
-2. This requirement is not final and can be changed in the future.
+# Requirement
+
+A datapack must satisfy these requirements to receive the Certified Datapack title
+
+- Datapack must be portable, it must not be tied down to a specific save.
+- Datapack must follow all of **Tier 3** Conventions.
+- Use `core`, `library`, `API`, `utils`, etc. These datapacks must also be certified.
+  - Your review status will be paused until those datapacks are also certified.
+- If you are not the owner, **don't `steal`, `copy`, or `claim`** that you created it.
+  - `Modification` or `further develop/expand/build on` should first check the **license** or **contact the owner**, and remember to **credit**.
```

Comment:
Poorly worded and excessive use of code block.

Reply:
```suggestion
- Use `core`, `library`, `API`, `utils`, etc. These datapacks must also be certified.
  - Your review status will be paused until those datapacks are also certified.
- You must be the owner or team leader of the datapack project. 
  - "Modification" or "further develop/expand/build on" should check **the license** or **contact the owner** first. And **don't forget to give credit**.
```